types/web3/web3-tests.ts

```diff
@@ -16,3 +16,8 @@ const myContract = new web3.eth.Contract(
 myContract.options.from = "0x1234567890123456789012345678901234567891";
 myContract.options.gasPrice = "20000000000000";
 myContract.options.gas = 5000000;
+
+const weiStr = web3.utils.toWei("100", "gwei");
+weiStr.endsWith(weiStr);
```

Comment:
What test is this?
It seems to me that is always true.
If you want to test 'gwei' runs correct, use same value as [this](https://web3js.readthedocs.io/en/1.0/web3-utils.html#id76).

Reply:
I guess this is a test whether or not `web3.utils.toWei` returns a string. This code will only be compiled, but not run.

Reply:
@Levino is correct, this is a compile-time test to check if returned value is a `BigNumber` or `string`.